### PR TITLE
Distinct Dropdown Values

### PR DESCRIPTION
**Pull request by philippdrebes**
Added the option to only show unique values in the dropdown menu.

### CODE REVIEW

File: src/autocomplete.js [hunk 1]
```diff
@@ -294,6 +294,16 @@
             const _this = this;
             this.destroyDropdownItems();
 
+            if (this.options.distinct) {
+                let distinct = [];
+                for (let item of dataItems) {
+                    if (!distinct.filter(x => x[this.options.nameProperty] === item[this.options.nameProperty]).length) {
```
Comment (florianbaer) [inline src/autocomplete.js]: Is this the solution with the best performance?

**Review by florianbaer — CHANGES_REQUESTED** [review]
Extend it with the HTML Attr distinct="true"

File: src/autocomplete.js [hunk 2]
```diff
@@ -168,6 +168,14 @@
             if (this.options.validation) {
                 this.options.validation = this.options.validation.bind(this);
             }
+
+            attr = this.$input.attr("distinct");
+            if (attr == 'false') {
```
Comment (florianbaer) [inline src/autocomplete.js]: Wieso == oder ===?

Comment (philippdrebes) [reply src/autocomplete.js]: verstehe die frage nicht

File: src/autocomplete.js (hunk 1 above)
Comment (philippdrebes) [reply src/autocomplete.js]: found a slightly faster alternative:
```shell
current x 45,904 ops/sec ±3.61% (85 runs sampled)
alternative x 50,582 ops/sec ±1.63% (89 runs sampled)
```

**Review by florianbaer — APPROVED** [review]
:shipit:

**Pull request merged**